internal/cmd/new.go

```diff
@@ -28,7 +30,7 @@ func (c *newScriptCmd) flagSet() *pflag.FlagSet {
 	return flags
 }
 
-func (c *newScriptCmd) run(_ *cobra.Command, args []string) error {
+func (c *newScriptCmd) run(_ *cobra.Command, args []string) (err error) {
```

Comment:
This can remove `nolint`s (`rerr` (return error) instead of `err`):

```suggestion
func (c *newScriptCmd) run(_ *cobra.Command, args []string) (rerr error) {
```

Reply:
Is this the case? I believe this is rather a linter issue. It claims that inside the `defer` block we assign a value that is never user afterwards, which is not true, because `err` will be returned upon the exit from `run()`. Or am I getting your suggestion wrong?

Reply:
My bad, the linter was complaining because `err` is being shadowed inside of the `if` block in the deferred function. I'll push a fix

Reply:
Shadowing always hunts us 😄 👻